src/translations/screens/subscreens/PaymentVipps.ts

```diff
@@ -2,27 +2,35 @@ import {translation as _} from '../../commons';
 
 const PaymentVippsTexts = {
   header: {
-    title: _('Videresendes til Vipps'),
+    title: _('Videresendes til Vipps', 'Redirecting to Vipps'),
     leftButton: {
-      a11yLabel: _('Avslutt Vipps-betaling og gå tilbake'),
+      a11yLabel: _(
+        'Avslutt Vipps-betaling og gå tilbake',
+        'Cancel Vipps payment and go back',
+      ),
     },
   },
   buttons: {
-    goToVipps: _('Gå til Vipps for betaling'),
-    tryAgain: _('Prøv igjen'),
-    goBack: _('Gå tilbake'),
+    goToVipps: _(
+      'Gå til Vipps for betaling',
+      'Go to Vipps service for payment',
```

Comment:
Kanskje vi skal si "app" i stedet for "tjeneste". Tenker i tilfellet turister o.l ikke forstår hva "Vipps" er.
```suggestion
      'Go to Vipps-app for payment',
```

Reply:
CC @magnusolderoesaether

Reply:
Har brukt Vipps service i andre oversetting. Jeg kan erstatte med "Vipps app"